storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -1177,25 +1177,22 @@ bool Redo_Log_Data_Manager::is_error() const { return (error); }
 Redo_Log_Data_Manager::~Redo_Log_Data_Manager() { os_event_destroy(event); }
 
 bool Redo_Log_Data_Manager::stop_at(lsn_t lsn) {
-  bool last_checkpoint = reader.find_last_checkpoint_lsn(&last_checkpoint_lsn);
```

Comment:
Why we moved this check? Any specific reason?

Reply:
reader.find_last_checkpoint_lsn does addition task to let pxb know if redo logs are disabled  if so we error out the backup.  That's why I didn't remove this check.